src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -3,31 +3,53 @@
 		class="kv-tailwind tw-w-[336px]"
 		:id="`${loanId}-loan-card`"
 	>
-		<!-- Borrower image -->
+		<!-- Borrower image w/location <summary-tag> -->
 		<kv-loading-placeholder
 			v-if="isLoading"
 			class="tw-mb-1 tw-rounded" :style="{width: '100%', height: '15.75rem'}"
 		/>
-		<borrower-image
+
+		<span
```

Comment:
I'd normally prefer a `div` for controlling layout of content items in this way.